examples/community/pipeline_stable_diffusion_3_ipa.py

```diff
@@ -0,0 +1,1403 @@
+# Copyright 2024 Stability AI, The HuggingFace Team and The InstantX Team. All rights reserved.
```

Comment:
Do we need this?

Reply:
That's the original implementation, we can remove